var/spack/repos/builtin/packages/py-netpyne/package.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyNetpyne(PythonPackage):
+    """Netpyne: A python package to facilitate the development,
+    parallel simulation, optimization and analysis of multiscale
+    biological neuronal networks in NEURON."""
+
+    homepage = "http://www.netpyne.org/"
+    url = "https://github.com/suny-downstate-medical-center/netpyne/archive/refs/tags/v1.0.3.1.tar.gz"
+    git = "https://github.com/suny-downstate-medical-center/netpyne.git"
+
+    version("master", git=git, branch="master")
```

Comment:
The `git` argument is not needed.

```suggestion
    version("master", branch="master")
```

Reply:
See example at https://spack.readthedocs.io/en/latest/packaging_guide.html#fetching-from-code-repositories.